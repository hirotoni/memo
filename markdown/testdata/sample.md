# heading 1

## unordered list

- list 1
  - indented 1
  - indented 2
- list 2
  - indented 1
- list 3
- list 4
  following list
  another following list
- list 5

## checkbox

- [ ] checkbox 1
- [ ] checkbox 2
- [x] checkbox 3
- [o] checkbox 4

# heading 1-2

### ordered list

1. ordered list 1
2. ordered list 2
3. ordered list 3
   1. indented ordered list
   1. indented ordered list
   1. indented ordered list

#### inline nodes

this is a paragraph.

this is a paragraph followed by
another paragraph without any blank lines in between.

this is **inline emphasis** in a paragraph.

- [link](destination/to/some/place)

[link](destination/to/some/place2)

##### heading 5

###### heading 6
