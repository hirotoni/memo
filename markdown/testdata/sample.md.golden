# heading 1

## unordered list

- list 1
  - indented 1
  - indented 2
- list 2
  - indented 1
- list 3
- list 4
  following list
  another following list
- list 5

## checkbox

- [ ] checkbox 1
- [ ] checkbox 2
- [x] checkbox 3
- [o] checkbox 4

## heading 2

below is thematic break

---

above is thematic break

### ordered list

1. ordered list 1
2. ordered list 2
3. ordered list 3
   1. indented ordered list
   2. indented ordered list
   3. indented ordered list

#### inline nodes

this is a paragraph.

this is a paragraph followed by
another paragraph without any blank lines in between.

this is **inline emphasis** in a paragraph.

this is `codespan`

- [link](destination/to/some/place)
- [link](destination/to/some/place)

1. [link](destination/to/some/place)
2. [link](destination/to/some/place)

[link](destination/to/some/place2)
[link](https://www.hirotoni.com)

autolink: https://www.hirotoni.com

##### heading 5

###### heading 6
